engines/suppliers/app/controllers/suppliers/api/v1/orders_frequencies_controller.rb

```diff
@@ -0,0 +1,77 @@
+module Suppliers
+  module Api
+    module V1
+      class OrdersFrequenciesController < ApplicationController
+        before_action :authenticate
+        before_action :load_orders_frequency, only: [:show, :update, :destroy]
+        before_action :load_group, only: [:index, :create]
+
+        # GET /api/v1/orders_frequencies?group_id=
+        #
+        # You must pass `group_id` param
+        #
+        def index
+          orders_frequencies = OrdersFrequency.where(group_id: @group.id)
+
+          render json: OrdersFrequenciesSerializer.new(orders_frequencies)
+        end
```

Comment:
What is the use case for showing all order frequencies at once? I thought this a one-to-one relation with group